### ISSUE TITLE
linux/arm64/v8 architecture support

### ISSUE THREAD

**Issue by GrahamCampbell** [haskell/docker-haskell#25]
Currently, this package is only built for `linux/amd64`. Is support for `linux/arm64/v8` planned? I understand GHC at least can be compiled on ARM64, now. I'm not sure what state it's actually in, in terms of actually working.

**Comment by AGSaidi** [comment]
Any plans to update the offical images to support arm64 (aarch64)? It would be great to see

**Comment by AlistairB** [comment]
Hi, initial arm64 support has been released!

Unfortunately [stack does not officially support arm64](https://github.com/commercialhaskell/stack/issues/2103) so is not included. There is a stack 2.7.1 arm64 release binary which I tried, but that didn't work in debian buster at least. I have pinged the stack people and there is interest in fixing this (although no movement yet), but I didn't want to wait.

It would be great if people could try out the arm64 images! :bow: 

cc @Bodigrim
